### ISSUE TITLE
chuaı: "frozen" vs "solid"

### ISSUE THREAD

**Issue by Ilmen-vodhr** [toaq/dictionary#70]
```
  {
    "toaq": "chuaı",
    "type": "predicate",
    "english": "▯ is frozen.",
    "gloss": "frozen",
    "keywords": [],
    "frame": "c",
    "distribution": "d",
    "notes": [],
    "examples": [],
    "fields": []
},
```
Can **chuaı** be used with the meaning "solid" (state of matter)?

**Comment by uakci** [comment]
I'd say yes.

**Comment by xorxes** [comment]
If that's the case, "frozen" is a bad gloss. Like "molten" or "vaporized", it indicates a different prior state.

**Comment by Ilmen-vodhr** [comment]
My rational for suggesting that **chuaı** could maybe be used for "solid" is that one could reasonably think that for example a rock is solid because it's "frozen", i.e. at a temperature blow that at which it would melt. That would be a "relative" view of the word "frozen", and not an absolute one based on the freezing temperature of water.

But on the other hand I suspect the English word "frozen" may imply the presence of frozen water in the solid material (e.g. "frozen soil"). Athough in astronomy one may talk about "frozen methane" for example, without implying the presence of water, if I'm not mistaken (please correct me if I am wrong there).

**Comment by Ilmen-vodhr** [comment]
So I suppose it should be clarified whether **chuaı** necessitates the presence of frozen water within the material.

**Comment by xorxes** [comment]
"Frozen lava" does not require water freezing temperatures. But it does require that the lava came from a previous non-solid state,

**Comment by Ilmen-vodhr** [comment]
Yeah. But maybe **chuaı** should not carry this indication of the previous state, as we might want to use root compounds for that (e.g. **loaılūıchūaı**, "liquid/flow-has_been-solid", or something like that). It might be preferable to have roots merely describe the current state of matter without any indication of the previous one, if any.

**Comment by xorxes** [comment]
My only objection was to the choice of English gloss word if it was meant for any solid.

A word like **chuaikea** would also need to be revised.

Maybe **loaisha**i for 'freezes' and **loaishailui** for 'frozen'
